tests/kafkatest/services/kafka/kafka.py

```diff
@@ -38,8 +38,7 @@
 class KafkaService(KafkaPathResolverMixin, JmxMixin, Service):
 
     PERSISTENT_ROOT = "/mnt"
-    STDOUT_CAPTURE = os.path.join(PERSISTENT_ROOT, "kafka.log")
-    STDERR_CAPTURE = os.path.join(PERSISTENT_ROOT, "kafka.log")
+    STDOUT_STDERR_CAPTURE = os.path.join(PERSISTENT_ROOT, "server-start-stdout-stderr.log")
```

Comment:
@ijuma: it captures whatever shows up in stdout and stderr while Kafka is running. Without this change, any SASL or SSL debugging messages are lost. I'm happy to rename the file if you think it's misleading.